src/lib/containers/download_list/DownloadConfirmation.tsx

```diff
@@ -120,64 +139,86 @@ const StatusConstruct: UiStateDictionary = {
 
 export const DownloadConfirmation: React.FunctionComponent<DownloadConfirmationProps> = ({
   getLastQueryRequest,
+  folderId,
   fnClose,
   updateParentState,
   topLevelControlsState,
   onExportTable,
+  downloadCartPageUrl='/DownloadCart',
 }) => {
   const { accessToken } = useSynapseContext()
   const { showDownloadConfirmation = true } = topLevelControlsState ?? {}
+  const [status, setStatus] = useState<StatusEnum>(accessToken ? StatusEnum.LOADING_INFO : StatusEnum.SIGNED_OUT)
   const [state, setState] = useState<DownloadConfirmationState>({
     fileCount: 0,
-    fileSize: 0,
-    status: accessToken ? StatusEnum.LOADING_INFO : StatusEnum.SIGNED_OUT,
+    fileSize: 0
   })
+  const [queryOrFolder] = useState<QueryOrFolder>({
+    queryBundleRequest: getLastQueryRequest!(),
```

Comment:
How come this one doesn't have a `setQueryOrFolder` variable?

Reply:
Right, it never updates this value.  I thought if I used a const then the object would be redefined on each render (which would cause the useEffect to fire).  This was my attempt to single object instance between renderings to avoid re-render (similar to why people use the useCallback() wrapper for inline functions).  Please let me know if there's a better way to do this!

Reply:
Ah, I see. That makes sense. Thanks.